### PR TITLE
refine case detail and study detail page's styling

### PR DESCRIPTION
**Pull request by jonkiky**

### CODE REVIEW

File: src/main/frontend/src/pages/caseDetail/caseDetailView.js [hunk 1]
```diff
@@ -334,7 +366,15 @@ const styles = (theme) => ({
     flexDirection: 'column',
   },
   caseCardContainer: {
-    marginTop: '32px',
+    padding: '16px',
+    background: '#f0f6f8',
```
Comment (AjayDoddapaneni) [inline src/main/frontend/src/pages/caseDetail/caseDetailView.js]: you can create and use a theme variable here.

File: src/main/frontend/src/pages/studyDetail/components/studyDetailView.js [hunk 2]
```diff
@@ -207,19 +209,18 @@ const styles = (theme) => ({
...
+    paddingBottom: '10px',
   },
   studyDetailButton: {
     float: 'right',
   },
   studyDetailContainer: {
-    maxWidth: '1000px',
-    margin: '16px auto',
+    padding: '16px',
+    background: '#f0f6f8',
```
Comment (AjayDoddapaneni) [inline src/main/frontend/src/pages/studyDetail/components/studyDetailView.js]: use the same theme variable here too.

File: src/main/frontend/src/pages/caseDetail/caseDetailView.js [hunk 3]
```diff
@@ -79,249 +165,195 @@ const CaseDetail = ({ classes, data }) => {
...
-                        <Grid item xs={12}>
-                          <Grid container spacing={8}>
-                            <Grid item xs={4}>
-                              <Typography weight="bold">Primary Site</Typography>
-                            </Grid>
-                            <Grid item xs={8}>
-                              <Typography>{diagnosis.primary_disease_site}</Typography>
-                            </Grid>
+                          <Grid item xs={8}>
+                            <Typography>{diagnosis.histology_cytopathology}</Typography>
```
Comment (AjayDoddapaneni) [inline src/main/frontend/src/pages/caseDetail/caseDetailView.js]: can you please add the error cases for all the elements

File: src/main/frontend/src/pages/caseDetail/caseDetailView.js (hunk 1 above)
Comment (jonkiky) [reply src/main/frontend/src/pages/caseDetail/caseDetailView.js]: Do we have the theme variables for them or I should create one?

**Pull request merged**